### PR TITLE
ICMSLST-1686 Use Section5Clause records when adding a section 5 goods line.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Changes:
  - Update section 5 form to use section 5 clauses in db.
  - Restrict fa-sil section 5 authorities using linked offices.
  - Restrict fa-sil verified authorities using linked offices.
  - Fix linked office list to only show active offices.
  - Fix fa-sil goods description sent to CHIEF

### CODE REVIEW

File: data_migration/tests/test_e2e.py [hunk 1]
```diff
@@ -374,7 +374,7 @@ def test_import_sil_data(mock_connect, dummy_dm_settings):
     assert web.SILGoodsSection2.objects.filter(**sil1_f).count() == 1
     assert web.SILGoodsSection5.objects.filter(**sil1_f).count() == 1
     sec5 = web.SILGoodsSection5.objects.filter(**sil1_f).first()
-    assert sec5.subsection == "Test Description"
+    assert sec5.section_5_clause.description == "Test Description"
     assert web.SILGoodsSection582Obsolete.objects.filter(**sil1_f).count() == 1  # /PS-IGNORE
```
Comment (MattHolmes123) [inline data_migration/tests/test_e2e.py:378]: Have you got a test for migrating the `clause` as well as the `description`?

File: web/domains/case/_import/fa_sil/widgets.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    def build_attrs(
+        self, base_attrs: dict[str, Any], extra_attrs: dict[str, Any] | None = None
+    ) -> dict[str, Any]:
+        attrs = super().build_attrs(base_attrs, extra_attrs)
+
+        attrs["data-minimum-input-length"] = 0
+        attrs["data-placeholder"] = "Please choose a subsection"
+
+        return attrs
```
Comment (chris-pettinga) [inline web/domains/case/_import/fa_sil/widgets.py:19]: Can't we pass these to the widget when we instantiate it?

File: web/domains/case/_import/fa/views.py [hunk 3]
```diff
@@ -323,15 +323,19 @@ def manage_certificates(request: AuthenticatedHttpRequest, *, application_pk: in
...
+            .filter(Q(linked_offices=None) | Q(linked_offices=application.importer_office))
+            .annotate(selected=selected_verified)
+        )
 
         extra_context = {}
 
         # FA-SIL specific context
         if application.process_type == ProcessTypes.FA_SIL:
-            verified_section5 = application.importer.section5_authorities.currently_active()
+            verified_section5 = application.importer.section5_authorities.currently_active().filter(
```
Comment (chris-pettinga) [inline web/domains/case/_import/fa/views.py:336]: is there scope to rename `currently_active` -> `active` so it's consistent with firearm authorities?

**Review by chris-pettinga — APPROVED** [review]

File: web/domains/case/_import/fa/views.py (hunk 3 above)
Comment (MattHolmes123) [reply web/domains/case/_import/fa/views.py:336]: Yer I thought that myself tbh 👍 Will take a look if the custom model manager is used in many places (I doub't it)

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**